### PR TITLE
improve docs

### PR DESCRIPTION
**Pull request by Jitendra-singh-123**

### CODE REVIEW

**Comment by Jitendra-singh-123** [comment]
please,can u accept my pull request?

File: docs/css/admonitions.css [hunk 1]
```diff
@@ -1,3 +1,4 @@
+                                                                  /* About CSS* */
```
Comment (StephenBrown2) [inline docs/css/admonitions.css:1]: I'm not sure how this improves the docs?

**Comment by Jitendra-singh-123** [comment]
Can u do it for me?
Please

**Comment by StephenBrown2** [comment]
Unfortunately, no, as I'm not a maintainer, and it's still unclear as to what this improves. You haven't provided a description in the PR, and the only change is an extremely indented comment.

**Comment by StephenBrown2** [comment]
This seems to be an `invalid`/`spam` PR, and I recommend closing it and tagging it with the aforementioned tags, as recommended by DigitalOcean's Hacktoberfest FAQ: https://hacktoberfest.digitalocean.com/details#quality

> Spammy pull requests can be given a label that contains the word "invalid" or "spam" to discount them. Maintainers are faced with the majority of spam that occurs during Hacktoberfest, and we dislike spam just as much as you. If you're a maintainer, please label any spammy pull requests submitted to the repositories you maintain as "invalid" or "spam", and close them. Pull requests with this label won't count toward Hacktoberfest.

**Pull request closed without merging**